src/briefcase/integrations/virtual_environment.py

```diff
@@ -203,15 +205,79 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         return False
 
 
+class NoOpVenvContext(Tool):
+    """Context for wrapping subprocess for no-op venv.
+
+    Provides the same interface as VenvContext but pass calls through to the native
+    subrpocess
+    """
+
+    name = "no_op_environment"
+    full_name = "No-Op Environment"
+
+    def __init__(self, tools, marker_path: Path, **kwargs):
+        super().__init__(tools, **kwargs)
+        self.created = False
+        self.venv_path = None
+        self.marker_path = marker_path
```

Comment:
Why not consolidate these two? The marker path is a known extension of the venv_path; might as well keep the interface to vent-like objects consistent, always create them with a path, and leave the construction of any internal marker files to the internal implementation.